src/container/RelayContainer.js

```diff
@@ -445,7 +438,7 @@ function createContainerComponent(
         }
         if (!deferredSubscriptions.hasOwnProperty(subscriptionKey)) {
           deferredSubscriptions[subscriptionKey] =
-            GraphQLDeferredQueryTracker.addListenerForFragment(
+            deferredQueryTracker.addListenerForFragment(
```

Comment:
To elaborate, this method alone requires `RelayContext.prototype.hasPendingQueries`, `RelayContext.prototype.isQueryPending` and `RelayContext.prototype.addListenerForFragment`.

Maybe `hasFragmentData` and the associated state/callbacks could be wrapped in core class and exposed through the public API with a single call...